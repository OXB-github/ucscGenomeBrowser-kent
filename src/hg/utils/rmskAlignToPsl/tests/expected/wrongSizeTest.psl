116	48	116	0	6	18	4	9	+	MamRep605b	901	692	874	chr1	248956422	6647872	6648045	11	17,5,7,11,53,12,9,12,16,15,7,	692,713,719,730,745,798,810,819,832,852,867,	6647872,6647889,6647894,6647901,6647912,6647966,6647981,6647991,6648003,6648019,6648038,
97	42	97	0	3	25	4	4	+	MamRep605b	901	662	826	chr1	248956422	13589311	13589454	8	35,32,19,14,15,6,13,5,	662,709,741,771,787,802,808,821,	13589311,13589346,13589379,13589398,13589412,13589428,13589435,13589449,
52	26	52	0	1	1	0	0	-	MamRep605b	901	42	121	chr1	248956422	13699382	13699460	2	19,59,	780,800,	13699382,13699401,
45	14	45	0	0	0	0	0	+	MamRep605b	901	707	766	chr1	248956422	16385363	16385422	1	59,	707,	16385363,
99	46	99	0	5	10	2	4	-	MamRep605b	901	492	647	chr1	248956422	19884198	19884347	8	45,4,19,17,7,6,29,18,	254,299,308,327,345,353,361,391,	19884198,19884246,19884250,19884270,19884287,19884294,19884300,19884329,
