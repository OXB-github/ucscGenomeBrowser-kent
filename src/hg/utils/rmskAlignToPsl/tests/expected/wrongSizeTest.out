Warning: 'FLAM_C_short_' size not found for query, skipping
